fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -999,5 +999,16 @@
      */
     @ConfField
     public static boolean check_java_version = true;
+
+    
+    @ConfField(mutable = true)
```

Comment:
```suggestion
    @ConfField(mutable = true, masterOnly = true)
```

Reply:
Also add comment for this config. Better to tell the user the best practice to set its value.